README.md

```diff
@@ -39,45 +31,24 @@ In addition to the steps above there are a few addition steps required for trans
 
 #### New Language
 
-1. Create a folder using the 2 character code (e.g. ja, en, es, etc) with lesson subfolders:
+1. Create a folder using the ISO language code (e.g. ja, zh-hans, es, et al) with lesson subfolders:
```

Comment:
we need to link to this page, and ask people to find the valid 2 letter, or 3 letter code.
https://www.loc.gov/standards/iso639-2/php/English_list.php